manager2/src/app/auth/register/register.component.html

```diff
@@ -18,19 +18,19 @@
                     <input autocomplete="family-name" (ngModelChange)="update_userid($event, 1)" id="lastname" [ngModelOptions]="{standalone: true}" [(ngModel)]="lastname" type="text" class="form-control" required>
                 </div>
                 <div class="col-md-4">
-                    <label for="email">User id</label>
+                    <label for="email">Username</label>
                     <input autocomplete="username" [ngModelOptions]="{standalone: true}" [(ngModel)]="userid" type="text" id="userid" class="form-control" required maxlength="12"/>
 
                 </div>
             </div>
             <div class="form-group">
-                <label for="email">Professional email</label>
+                <label for="email">Professional academic email</label>
```

Comment:
why academic ? We may have non academic

Reply:
you're right 74b9261e5120bfcd165953309a9d4ce535ef70ca